components/dashboard/src/projects/Project.tsx

```diff
@@ -171,10 +171,10 @@ export default function () {
                     return <Item key={`branch-${index}-${branchName}`} className="grid grid-cols-3 group">
                         <ItemField className="flex items-center">
                             <div>
-                                <div className="text-base text-gray-900 dark:text-gray-50 font-medium mb-1">
+                                <a href={branch.url}><div className="text-base text-gray-900 dark:text-gray-50 font-medium mb-1">
```

Comment:
**thought**: I'd love to also link to each commit from the commit hash in the column as described in the relevant issue in https://github.com/gitpod-io/gitpod/issues/5585. However, we can skip if for this PR and open a follow-up issue for this. 🏓

Reply:
**fyI**: Added follow up issue in https://github.com/gitpod-io/gitpod/issues/5923.